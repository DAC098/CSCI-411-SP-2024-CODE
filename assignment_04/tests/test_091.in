5
-3 2 2
gqmlxkkiizhnl uprumgtnbtmpsmzmktpyvqbku
aataca tccccg
ffrolqknflkvvxschpnqufikmcojdidcmdqbvr kopyiodkljpqgouwxutzkb
ccc aatatta
ocnuusjizidzummggqmubvxcduhhjyu uziztwvehdlqjlhfmj
