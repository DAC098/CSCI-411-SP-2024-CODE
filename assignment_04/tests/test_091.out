_________________________gqmlxkkiizhnl
uprumgtnbtmpsmzmktpyvqbku_____________
-49
______aataca
tccccg______
-6
______________________ffrolqknflkvvxschpnqufikmcojdidcmdqbvr
kopyiodkljpqgouwxutzkb______________________________________
10
_______ccc
aatatta___
-15
__________________ocnuusjizidzummggqmubvxcduhhjyu
uziztwvehdlqjlhfmj_______________________________
8
